src/components/SkillsPanel.js

```diff
@@ -72,6 +72,15 @@ export default function SkillsPanel({ characterStats }) {
 }
 
 function SkillTile({ skill, selectedSkill, setSelectedSkill, unlockedSkills, level = 1 }) {
+    const dispatch = useDispatch();
+    const isSkillUnlocked = unlockedSkills.includes(skill);
+    const isDefaultSkill = DEFAULT_UNLOCKED_SKILLS.includes(skill);
+
+    useEffect(() => {
+        if (level > 1 && !isSkillUnlocked) dispatch(unlockSkill({ skill }));
+        if (level === 1 && isSkillUnlocked && !isDefaultSkill) dispatch(lockSkill({ skill }));
```

Comment:
it's so weird how there was a ghost replying to you earlier, huh.
Anyway uh... 
Yeah I think for now let's just assume there's only one character, since all the other character state like tasks is global. Multi-char support will come w/backend (prob this weekend)

Reply:
Are we forcing users to log in and have account links? What if someone just wanted to open the site and look up a bunch of users?

Reply:
I'm thinking I'll still have a multi-character option for non-account users. It'll be configured in some kind of character switcher interface though. There wouldn't be much benefit to looking up characters other than your own I think since it just gives you their hiscores, which you can get directly from jagex.

Read-only views (maybe through sharable permalinks) of other characters' stats/tasks/relics/etc are something I'm considering for that type of usecase later though.

Reply:
I guess what we really want is for the unlocks etc to reset to base when a new user is used (highscore lookup, multicharacter). Should probably move the unlock logic to some kind of "new accont used - process unlocks" method

What do you want to do with this PR?

Reply:
yep a lot of refactoring will need to happen for multi-character to work. I think just drop the auto-relock at level 1 line, then keep the rest of the PR. Unlocking automatically when you've leveled the skill is a nice convenience thing so def keep that